doc/integrations/pandas.md

```diff
@@ -95,3 +95,10 @@ will create a table name in the database from the named DataFrame.   Or use `--a
 ```{code-cell} ipython3
 
 ```
+
+````
+The `--persist-replace` argument, with the name of a  DataFrame object in memory,
+will replace the existing name DataFrame object in memory.
+```{code-cell} ipython3
```

Comment:
I think you should add a new line between the text and the {code-cell} so the code cell takes effect (and remove the extra ```)


![image](https://user-images.githubusercontent.com/103026689/226338961-886b985d-1ae9-40c6-998c-9e2616dff128.png)